buildSrc/src/main/kotlin/pklJavaLibrary.gradle.kts

```diff
@@ -19,6 +19,7 @@ import org.gradle.accessors.dm.LibrariesForLibs
 
 plugins {
   `java-library`
+  id("pklJvmToolchain")
```

Comment:
all `pklJavaLibrary` and `pklKotlinLibrary` projects now automatically pick up `pklJvmToolchain`